src/css/index.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
 * {
   padding: 0;
   margin: 0;
```

Comment:
Hi Team, good job. I think would be better if you fix the linter errors in order to  make your project more organize and problems free, kindly consider fixing this and everything is done.